docs/qprofiler2/index.md

```diff
@@ -0,0 +1,328 @@
+
+## Usage
+
+~~~~{.text}
+java -jar qprofiler2.jar -h
+usage: java -jar qprofiler2.jar --input <input file> --output <output xml file> --log <log file> [options]
+
+Option                        Description                                                               
+------                        -----------                                                               
+--bam-full-header             Opt, this option indicate qprofiler2 output entire BAM header ; otherwise 
+                                only output HD and SQ.                                                  
+--bam-index                   Opt, a BAI index file. The option of "threads-producer" will be ignored   
+                                if no index file provided.                                              
+--bam-records <Integer>       Opt, only process the specified number of records from the beginning of   
+                                the BAM file. Default to process entire BAM file.                       
+--bam-validation              Opt, BAM record validation stringency [STRICT,LENIENT,SILENT].            
+                                Def=LENIENT.                                                            
+--help                        Show usage and help.                                                      
+--input                       Req, input file in FASTQ, BAM/SAM or VCF format, also support FA, MA,     
+                                QUAL, FASTA, GFF and GFF3 format.                                       
+--log                         Req, Log file.                                                            
+--loglevel                    Opt, Logging level [INFO,DEBUG], Def=INFO.                                
+--output                      Opt, XML output which containing basic summary statistics. Def="qprofiler.
+                                xml".                                                                   
+--threads-consumer <Integer>  Opt, number of threads to process inputed records, BAM and FASTQ mode     
+                                only. Def=0 (not multi-thread mode).                                    
+--threads-producer <Integer>  Opt, number of threads to read the indexed BAM file, BAM mode only. This  
+                                option will be ignored without "threads-consumer" option.  Def=1.       
+--vcf-format-field            Opt, group VCF records according to user specified format fields. eg.     
+                                "GD", "CCM","FT=PASS". Default to ignore VCF format field.              
+--version                     Show version number.     
+~~~~
+
+
+### multi thread
+When running multi-threaded, we suggest more consumer than producer threads with a recommended ratio of 6:1. However, it is up to your machine system. for example, only one thread to read the input file but 12 threads are specified to process reads.
```

Comment:
Capital letter after a full stop.
"machine system"???
The last sentence doesn't make sense.